modules/openy_features/openy_prgf/modules/openy_prgf_branches_popup_class/openy_prgf_branches_popup_class.info.yml

```diff
@@ -3,8 +3,10 @@ description: 'OpenY Paragraph Branches Popup (Class).'
 type: module
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
 dependencies:
   - field
   - openy_prgf
   - paragraphs
   - plugin
+  - openy_popups
```

Comment:
Is it missed dependency?

Reply:
yes, this paragraph use block from `openy_popups` module, looks like on feature export it was ignored, so I decide to add this